src/classes/RLLP_OppPartialSoftCreditRollup.cls

```diff
@@ -164,6 +166,18 @@ public class RLLP_OppPartialSoftCreditRollup {
 
         UTIL_DMLService.updateRecordBatchAndLogErrors(toUpdate, ERR_Handler_API.Context.RLLP.name());
     }
+
+    /*******************************************************************************************************
+    * @description Checks if the Opportunity should be considered as Hard Credit.
+    * @param OpportunityContactRole The Opportunity Contact Role.
+    * @param Boolean The Always Rollup Primary Contact settings.
+    * @return Boolean.
+    */ 
+    private static Boolean isHardCredit(OpportunityContactRole ocr, Boolean alwaysRollupToPrimaryContact) {
```

Comment:
I wouldn't include the alwaysRollupToPrimaryContact parameter, I'd just look at the settings value directly inside the method.

Also, for clarity in the boolean logic, this is how I'd write it:

`if (UTIL_CustomSettingsFacade.getHouseholdsSettings().npo02__Always_Rollup_to_Primary_Contact__c) {
    return ocr.IsPrimary;
} else {
    //primary OCR is a hard credit only for individual donations
    return ocr.IsPrimary && ocr.Opportunity.Account.npe01__SYSTEMIsIndividual__c;
}`